open-build-service/notqmail.spec.in

```diff
@@ -0,0 +1,365 @@
+                                  %{_unitdir}/%{name}.service
+%endif
+
+%if 0%{?suse_version} >= 1120
+%attr(644,root,root) %config(noreplace) %{_sysconfdir}/permissions.d/%{name}-permissions
+%attr(644,root,root) %config(noreplace) %{_sysconfdir}/permissions.d/%{name}-permissions.secure
+%verify (not user group mode caps) %attr(4755, qmailq, qmail)   %{qmaildir}/bin/qmail-queue
+%else
+%attr(4755,qmailq,qmail)           %{qmaildir}/bin/qmail-queue
+%endif
+
+%files doc
+%defattr(-, root, root,-)
+%dir %attr(0755,root,root) %{nq_docdir}
+     %attr(0644,root,root) %{nq_docdir}/*
+%attr(0644,root,root) %{_mandir}/man?/*%{?ext_man}
+
+### SCRIPTLETS ###############################################################################
+%verifyscript
+%if 0%{?suse_version} >= 1120
+  %verify_permissions -e %{qmaildir}/bin/qmail-queue
+  %verify_permissions -e %{qmaildir}/alias/
+  %verify_permissions -e %{qmaildir}/queue/lock/trigger
+%endif
+/var/qmail/bin/instcheck
+
+%pretrans
+argv1=$1
+if test -f %{_sysconfdir}/systemd/system/multi-user.target.wants/%{name}.service
+then
+  echo "Giving %{name} exactly 5 seconds to exit nicely"
+  /bin/systemctl stop %{name} > /dev/null 2>&1
+fi
+sleep 5
+
+%if 0%{?fedora_version} || 0%{?centos_version} || 0%{?rhel_version} || 0%{?suse_version} < 1500 || 0%{?sles_version} < 15
+%pre
+  /usr/bin/getent group nofiles   > /dev/null || /usr/sbin/groupadd nofiles
+  /usr/bin/getent group qmail     > /dev/null || /usr/sbin/groupadd qmail
+  /usr/bin/getent passwd alias    > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{qmaildir}/alias  -s /sbin/nologin alias
+  /usr/bin/getent passwd qmaild   > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{qmaildir}        -s /sbin/nologin qmaild
+  /usr/bin/getent passwd qmaill   > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{logdir}          -s /sbin/nologin qmaill
+  /usr/bin/getent passwd qmailp   > /dev/null || /usr/sbin/useradd -M -g nofiles  -d %{qmaildir}        -s /sbin/nologin qmailp
+  /usr/bin/getent passwd qmailq   > /dev/null || /usr/sbin/useradd -M -g qmail    -d %{qmaildir}        -s /sbin/nologin qmailq
+  /usr/bin/getent passwd qmailr   > /dev/null || /usr/sbin/useradd -M -g qmail    -d %{qmaildir}        -s /sbin/nologin qmailr
+  /usr/bin/getent passwd qmails   > /dev/null || /usr/sbin/useradd -M -g qmail    -d %{qmaildir}        -s /sbin/nologin qmails
+for i in alias qmaild qmaill qmailp qmailq qmailr qmails
+do
+  %{__rm} -f /var/spool/mail/$i
+done
+%else
+%pre -f notqmail.pre
+%endif
+%if 0%{?suse_version} || 0%{?sles_version}
+  %service_add_pre %{name}.service
+%endif
+
+### SCRIPTLET ###############################################################################
+%post
+%if 0%{?suse_version} >= 1120
+%if 0%{?set_permissions:1} > 0
+  %set_permissions %{qmaildir}/bin/qmail-queue
+  %set_permissions %{qmaildir}/alias/
+%else
+  %run_permissions
+%endif
+%endif
+/var/qmail/bin/instpackage queue-only
+/var/qmail/bin/instchown queue-only
```

Comment:
That will break as they are in libexec, no?

Reply:
good catch - fixed. Will get updated in the next commit.

Reply:
This has been addressed in the latest commit #[3eaf213](https://github.com/notqmail/notqmail/pull/76/commits/3eaf2138ce40da0831bfc90852faa8b7c2a2c4ff)